src/components/10-atoms/text/README.md

```diff
@@ -2,16 +2,51 @@
 
 A convenience component to apply possible AXA text styles to a piece of text.
 
-It accepts simple text strings as children or top-level HTML tags like `<p>` or `<span>`. Nested HTML tags are not supported.
+It accepts simple text strings that get rendered in a paragraph tag or if you want to render text nodes wrapped in custom tags. You should use the attribute `tagless` so that the wrapping paragraph is not rendered.
```

Comment:
Revert to previous text, which was better and covered more cases. Then _add_ 1 sentence like:

The default behaviour of wrapping simple text in a p(aragraph) text can be suppressed via the Boolean attribute `nowrap`.